### PR TITLE
Fix  syslog using TCP and UDP warning  callback

### PR DESCRIPTION
**Pull request by Rebits**
This PR fix error in some` test_basic_configuration_connection_valid` test cases

### CODE REVIEW

File: deps/wazuh_testing/wazuh_testing/remote.py [hunk 1]
```diff
@@ -154,12 +154,13 @@ def callback_warning_syslog_tcp_udp():
     Returns:
         callable: callback to detect this event.
     """
-    msg = fr"WARNING: \(\d+\): Only secure connection supports TCP and UDP at the same time. \
-          Default value \(TCP\) will be used."
+    msg = fr"WARNING: \(\d+\): Only secure connection supports TCP and UDP at the same time. " \
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/remote.py:157]:
```suggestion
    msg = r"WARNING: \(\d+\): Only secure connection supports TCP and UDP at the same time. " \
```

File: deps/wazuh_testing/wazuh_testing/remote.py [hunk 2]
```diff
@@ -154,12 +154,13 @@ def callback_warning_syslog_tcp_udp():
...
+          fr"Default value \(TCP\) will be used."
```
Comment (jmv74211) [inline deps/wazuh_testing/wazuh_testing/remote.py:158]:
```suggestion
          r"Default value \(TCP\) will be used."
```

File: deps/wazuh_testing/wazuh_testing/remote.py (hunk 1 above)
Comment (Rebits) [reply deps/wazuh_testing/wazuh_testing/remote.py]: Done ce6f11f4a48ef4e10aa074b93b752d882aeb996c

File: deps/wazuh_testing/wazuh_testing/remote.py (hunk 2 above)
Comment (Rebits) [reply deps/wazuh_testing/wazuh_testing/remote.py]: Done ce6f11f4a48ef4e10aa074b93b752d882aeb996c

**Review by jmv74211 — DISMISSED** [review]

**Review by BraulioV — APPROVED** [review]

**Pull request merged**